core/registry.ts

```diff
@@ -61,19 +61,44 @@ export const DEFAULT = 'default';
  * @alias Blockly.registry.Type
  */
 export class Type<T> {
+  // Type.CONNECTION_CHECKER = new Type('connectionChecker');
```

Comment:
I'm not quite sure how we ended up her, but I can't really see point keeping these comments.

Though I think I'd move the `static` declarations below the non-static methods (i.e., to where these comments were previously) if possible, as I think it's slightly clearer to declare the type  before declaring all the instances.